README.md

```diff
@@ -1,3 +1,106 @@
+
+<a name="how-dose-mirkos-work"></a>
+## How Does MIKROS Work?
+MIKROS sorts users by spending and activity levels in its ecosystem. This differs from how data is currently managed by competitors as they do not offer any data pooling. Game developers and advertisers can then view reports in a dashboard with valuable information which is needed to identify areas to improve their customer experience. With MIKROS, advertisers can now be certain that their ads are being shown to verified spenders.
+
+<a name="how-to-implement-mirkos"></a>
+## How to Implement MIKROS?
+1. In order to use Mikros, clients (game developers/publishers) must go to https://developer.tatumgames.com/
+2. Select a subscription plan. The options are FREE, STARTUP, ENTERPRISE
+3. Register an account, and verify this account via email
+4. Once inside the Mikros dashboard, the client can "create app". The process of creating an app is filling out a form that includes details such as company name, game title, description, package name (bundle id), product assets, videos, category {RPG, Puzzle, Platformer, CCG, Strategy, Other}
+
+Once steps 1-4 is completed, the following will be generated:
+1. App Game ID (auto-generated)
+2. Production API Key (auto-generated)
+3. QA API Key (Manual)
+4. Development API Key (Manual)
+
+Now, the client has to include the Mikros SDK into their gaming product and setup the SDK by the following method:
+1. In the Unity Editor, select **Mikros > Edit Settings**
+2. In the Inspector tab of **Mikros Settings**, paste in your App Game ID and API keys that have been generated.
+3. *(Optional)* If you want manual initialization of the SDK, disable the ***Auto Initialize At Start*** option from the Mikros Settings. By default, it is kept enabled.
+4. *(Optional)* You also have the option to select which API key to use at any development stage. By default, Production API Key is used.
+5. *(Optional)* Mikros collects and sends session data for better user insights and is also crucial for Analytics purpose. This can be enabled or disabled by the ***Auto Track User Session*** option from the Mikros Settings. By default, it is kept enabled and is recommended to keep it that way.
+6. *(Optional)* Mikros collects and sends metadata for better user insights and is also crucial for Analytics purpose. This can be enabled or disabled by the ***Auto Track User Metadata*** option from the Mikros Settings. By default, it is kept enabled and is recommended to keep it that way.
+
+<a name="sample-app-&-documentation"></a>
+## Sample App & Dcumentation
+
+* Authentification
+    
+    
+ This is the Mikros Sample App main screen. The Authentication button will allow you to be authenticated and allow you to use the Mikros Sample App.
```

Comment:
@noah1013 @Blu3m00n11 @drxeno02 The integrators won't be provided with our sample app. So anything related to it should be removed.

Reply:
@Debadri-Redapple Yes, exactly. I was going to redo this README after they submit. This should be a guide on how to use MIKROS. I think I will have to be the one to finalize everything. For now, the work here is about 90% complete. I will finish everything else off.

We'll let this PR sit until I can get to it.

Reply:
@drxeno02 This Sample App & Documentation section needs to be removed from Readme, before we can merge this PR.